lib/librte_hash/rte_cuckoo_hash.c

```diff
@@ -1160,3 +1160,63 @@ rte_hash_iterate(const struct rte_hash *h, const void **key, void **data, uint32
 
 	return position - 1;
 }
+
+inline int
+rte_hash_get_num_buckets(struct rte_hash *h)
+{
+	RETURN_IF_TRUE((h == NULL), -EINVAL);
+	return h->num_buckets;
+}
+
+inline uint32_t
+rte_hash_get_primary_bucket(const struct rte_hash *h, hash_sig_t sig)
```

Comment:
`rte_hash_get_primary_bucket()` is also exported in the header file, so it must move there.